warehouse/templates/packaging/detail.html

```diff
@@ -134,6 +134,32 @@ <h3 class="sidebar-section__title">Project Links</h3>
         </div>
         {% endif %}
 
+        {% if release.github_repo_info_url %}
+        <div class="sidebar-section github-repo-info hidden" data-url="{{release.github_repo_info_url}}">
+          <h3 class="sidebar-section__title">GitHub stats</h3>
```

Comment:
Please capitalize and use full word "Statistics" rather than "Stats"

i.e. "GitHub Statistics"